### ISSUE TITLE
Good correlation not causation example: blood pressure and surgery complications

### ISSUE THREAD

**Issue by nickeubank** [nickeubank/ds4humans#9]
Suppose high blood pressure predicts complications. Maybe it's the high blood pressure itself, maybe people with three jobs who take public transit are just leading more stressful lives. 

So blood pressure is great predictor of complications; but would it respond to blood pressure drugs? 

(And does that matter if just targeting for followup care?)

**Comment by joshclinton** [comment]
Suppose drinking wine predicts high income.  Does drinking more wine lead to more income?  

Or drinking wine related to healthy hearts (see this all the time and could easily find citations about this).  So does drinking more lead to healthier heart?  Or is it that those who drink wine are also likely to be richer, see doctors, exercise, eat healthy, etc.

**Comment by nickeubank** [comment]
Yup! But also want examples where the simple correlation may be useful for some purposes, even if not causal.

**Comment by joshclinton** [comment]
The umbrella example is a good one.  We can always build on this later, but if they don't get the initial idea the whole thing is moot....
